drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -583,6 +609,31 @@ func extractPodUID(volDirPath string) string {
 	return ""
 }
 
+// return PX node on k8s cluster provided by kubeconfig file
+func getPXNode(destKubeConfig string) (corev1.Node, error) {
+	var workerNode corev1.Node
+	// get schd-ops/k8s instance of destination cluster
+	destClient, err := k8s.NewInstance(destKubeConfig)
+	if err != nil {
+		logrus.Errorf("Unable to get k8s instance: %v", err)
+		return workerNode, err
+	}
+	nodes, err := destClient.GetNodes()
+	if err != nil {
+		return workerNode, err
+	}
+
+	// get label on node where PX is Enabled
+	for _, node := range nodes.Items {
+		if node.Labels[pxEnabled] == "true" {
+			workerNode = node
```

Comment:
You can do `return node, nil` here. And at the end of the function return an error since that will be reached if no node was found.